config/storage.yml

```diff
@@ -7,12 +7,12 @@ local:
   root: <%= Rails.root.join("storage") %>
 
 # Use rails credentials:edit to set the AWS secrets (as aws:access_key_id|secret_access_key)
-# amazon:
-#   service: S3
-#   access_key_id: <%= Rails.application.credentials.dig(:aws, :access_key_id) %>
-#   secret_access_key: <%= Rails.application.credentials.dig(:aws, :secret_access_key) %>
-#   region: us-east-1
-#   bucket: your_own_bucket
+amazon:
+  service: S3
+  access_key_id: <%= Figaro.env.AWS_ENV_ACCESS_KEY_ID %>
+  secret_access_key: <%= Figaro.env.AWS_SECRET_ACCESS_KEY %>
+  region: <%= Figaro.env.AWS_REGION %>
+  bucket: caucus_<%= Rails.env %>
```

Comment:
FWIW I just checked and our actual S3 bucket is `caucus-dev`, which is a different delimiter (hyphen instead of underscore) and also not the correct `Rails.env` value. Was this working for you?

Reply:
Fixed this up.